tapMainExperiment.m

```diff
@@ -180,7 +157,7 @@
         expParam.data(seqi).seq.outAudio = [];
```

Comment:
Due to structure array, I have to use  `expParam.data.seq(seqi).outAudio = []; `
But please do check if you wanted to keep outAudio somewhere and it's not saved in. I do not think so because in every sequence loop you are deleting it.